thorlcr/master/thgraphmanager.cpp

```diff
@@ -710,12 +710,13 @@ void CJobManager::run()
             break;
         }
         StringAttr graphName, wuid;
-        const char *wuidGraph = item->queryWUID(); // actually <wuid>/<graphName>
+        const char *wuidGraph = item->queryWUID(); // actually <wfid>/<wuid>/<graphName>
```

Comment:
May be worth renamed wuidGraph to jobInfo (or jobName) to avoid confusion in the future?

Reply:
minor, but yes I agree, but not worth changing now (merged)